pyvista/ext/plot_directive.py

```diff
@@ -55,7 +55,7 @@
         directives for which the ``:context:`` option was specified.  This only
         applies to inline code plot directives, not those run from files.
 
-    nofigs : bool
+    nofigs : None
```

Comment:
ref https://numpydoc.readthedocs.io/en/stable/format.html#parameters
```suggestion
    nofigs : bool, default None
```